src/Main.java

```diff
@@ -2,14 +2,40 @@
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-        //TIP Press <shortcut actionId="ShowIntentionActions"/> with your caret at the highlighted text
-        // to see how IntelliJ IDEA suggests fixing it.
-        System.out.printf("Hello and welcome!");
 
-        for (int i = 1; i <= 5; i++) {
-            //TIP Press <shortcut actionId="Debug"/> to start debugging your code. We have set one <icon src="AllIcons.Debugger.Db_set_breakpoint"/> breakpoint
-            // for you, but you can always add more by pressing <shortcut actionId="ToggleLineBreakpoint"/>.
-            System.out.println("i = " + i);
+        int[] sum = {123, 202, 404, 673, 900};
+        int total = 0;
+        for (int current : sum) {
+            total = total + current;
+        }
+        System.out.println("Сумма трат за месяц составила " + total + " рублей");
+
+
+        int[] secondSum = {243, 1149, 4909, 129, 590};
+        int max = -1;
+        int min = Integer.MAX_VALUE;
+        for (int i = 0; i < secondSum.length; i++) {
+            if (secondSum[i] > max) {
+                max = secondSum[i];
+            }
+            if (secondSum[i] < min) {
+                min = secondSum[i];
+            }
+        }
+        System.out.println("Минимальная сумма трат за неделю составила " + min + " рублей");
+        System.out.println("Максимальная сумма трат за неделю составила " + max + " рублей");
+
+        int[] thirdSum = {490, 643, 530, 430, 875};
+        total = 0;
+        for (int current : thirdSum) {
+            total = total + current;
+        }
+        double mean = (double) total / 5;
```

Comment:
```suggestion
        double mean = (double) total / thirdSum.lenth;
```
делить нужно на длину массива, а не число "с потолка"